TeamZeroServer/README_server.txt

```diff
@@ -0,0 +1,40 @@
+Once a connection with a client is established, the server can receive string text messages from them. These message are expected to be in JSON format, with one of the
+following types:
+
+To register with the server for the first time:
+	{
+		type: "REGISTER"
+		username:
+		password:
+		email:
+		picture:
```

Comment:
Should the user select the profile picture right at registration? Or is it better to just use the "edit profile" JSON down to change the picture from a default?